css/store.css

```diff
@@ -0,0 +1,238 @@
+/* store page */
+
+/* main */
+.storemain {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+/* section one */
+.formsection {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  gap: 30px;
+}
+/* form */
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+/* form title */
+.formsection h1 {
```

Comment:
It's better to add a class to the `h1` and target that directly in your CSS. That way, if the `h1` is later moved outside the container, the styling will still be applied